README.md

```diff
@@ -0,0 +1,73 @@
+# Key Broker Server for SEV
+
+A Key Broker Server (KBS) or Attestation Server verifies the launch measurement of a secure guest and conditionally provides secrets.
+
+This KBS currently supports SEV(-ES) with pre-attestation. 
+The KBS has two gRPC endpoints, `GetBundle` and `GetSecret`. 
+These endpoints are described below and defined in the [protobuf](srcr/grpc/keybroker.proto).
+
+To validate confidential guests, the guest owner must pre-provision the KBS
+with policy information. This KBS uses a database to store policies and secrets.
+The database is described below.
+
+## API
+
+### GetBundle
+The `GetBundle` endpoint taks a policy and certificate chain.
+After verifying the certificate chain, the KBS provides a session file and guest owner public key.
+This session file defines a secure channel between the KBS and the PSP, which will be used to securely provide secrets.
+
+### GetSecret
+The `GetSecret` endpoint takes the launch measurement of the guest, the expected launch parameters, and a secret request. 
+The expected launch parameters such as policy and firmware version are used to select the appropriate owner-approved launch digest. 
+The cryptographically verified launch measurement guarantees the correctness of the launch parameters. 
+
+The secret request allows the user to request multiple secrets in different formats. 
+With an offline KBC, secrets are injected ahead of time. Thus, requesting multiple secrets at once is essential.
+The JSON bundle secret format is compatible with the Attestation Agent's `offline-sev-kbc`.
+The secrets are provided in the OVMF secret table format.
+
+Additional secret types may be supported in the future.
+
+
+## Policies and Secrets 
+
+In SEV terms the policy is a group of flags that specifies properties of a confidential guest. 
+Here the policy also refers more generally to pre-provisioned guidelines that the KBS uss to evaluate the launch measurement of a VM and to determine whether a secret should be injected.
+By default the KBS enforces one tenant-wide policy, which is specified in `default_policy.json`.
+```
+{
+  "allowed_digests": [],
```

Comment:
As written, this suggests that the standard `default_policy.json` does not accept anything, since there are no allowed digests or policies. Is this intentional?

Reply:
If you want a default policy that denies all requests, you can just turn off the KBS. Alternatively, you can just set one allowed digest that is basically any random string. 

That said, I get that it is a little weird to have the empty set stand for anything. I guess we could add an explicit wildcard it would just make the parsing a bit more complex.